### DISCUSSION TITLE
Document `{ column='geom', type=* }`

### DISCUSSION THREAD

**Discussion by tordans** [osm2pgsql-dev/osm2pgsql#1844] [Q&A] [answered]
I am looking for the docs that tell which values the `type` can have for `{ column='geom', type=* }`.

I see https://osm2pgsql.org/doc/manual.html#geometry-types, but that has camel case names so I assume this is not the right docs section for this case.

I found https://github.com/openstreetmap/osm2pgsql/blob/a0ae47c63f3f398f3cc72d4308c42b7cf0e43f95/flex-config/bbox.lua#L17 which shows those values in use

* `{ column = 'geom', type = 'point' }`
* `{ column = 'geom', type = 'linestring' }`
* `{ column = 'geom', type = 'geometry' }`
* `{ column = 'geom', type = 'multilinestring' }`

In https://osm2pgsql.org/doc/manual.html#the-flex-output the `define_table(OPTIONS)` references `OPTIONS` but I think there is no place that would show what this `OPTIONS` can be, is there? That would likely be a good place to document the values(?).

**Comment by lonvia** [comment, marked as answer]
Types for flex output columns are described in [Defining Columns](https://osm2pgsql.org/doc/manual.html#defining-columns). There is an extra [subsection for geometries](https://osm2pgsql.org/doc/manual.html#defining-geometry-columns).

**Reply by tordans** [reply]
Thanks for the help!
